g2g_test.go

```diff
@@ -84,22 +85,22 @@ func TestRoundFloat(t *testing.T) {
 //
 
 type MockGraphite struct {
-	t     *testing.T
-	port  int
-	count int
-	mtx   sync.Mutex
-	ln    net.Listener
-	done  chan bool
+	t       *testing.T
+	address string
+	count   int
+	mtx     sync.Mutex
+	ln      net.Listener
+	done    chan bool
 }
 
-func NewMockGraphite(t *testing.T, port int) *MockGraphite {
+func NewMockGraphite(t *testing.T, address string) *MockGraphite {
 	m := &MockGraphite{
-		t:     t,
-		port:  port,
-		count: 0,
-		mtx:   sync.Mutex{},
-		ln:    nil,
-		done:  make(chan bool, 1),
+		t:       t,
+		address: address,
```

Comment:
Better to have network, address in the struct and split it during construction, than to split in loop, I think.